src/components/Shortcuts/ClipboardShortcuts/ClipboardShortcuts.tsx

```diff
@@ -61,19 +62,23 @@ export function ClipboardShortcuts() {
         return;
       }
 
-      const activeElementId = slideInstance.getActiveElementId();
+      const activeElementIds = slideInstance.getActiveElementIds();
 
-      if (!activeElementId) {
+      if (!activeElementIds.length) {
         return;
       }
 
-      const activeElement = slideInstance.getElement(activeElementId);
+      const orderedActiveElementIds =
+        slideInstance.sortElementIds(activeElementIds);
+      const activeElements = slideInstance.getElements(orderedActiveElementIds);
 
-      if (!activeElement) {
+      if (isEmpty(activeElements)) {
```

Comment:
`activeElements` have a type we can just use to check if it is empty without using external function
```suggestion
      if (Object.keys(activeElements).length === 0) {
```